concepts/05 Widgets/DataGrid/20 Editing/20 API/30 Delete.md

```diff
@@ -61,7 +58,87 @@ Call the [deleteRow(rowIndex)](/api-reference/10%20UI%20Widgets/GridBase/3%20Met
         ],
         // ...
     })
-    
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid
+            ref="DataGrid">
```

Comment:
Write ref names is lowerCamelCase. Please make this change in all these code samples.